Web Server consolidation to just NGINX with uWSGI

```diff
@@ -0,0 +1,207 @@
+        proxy_set_header X-Forwarded-For $remote_addr;
+        proxy_set_header Host $host;
+        proxy_cache cache;
+        proxy_cache_bypass $cookie_auth_tkt;
+        proxy_no_cache $cookie_auth_tkt;
+        proxy_cache_valid 30m;
+        proxy_cache_key $host$scheme$proxy_host$request_uri;
+        # In emergency comment out line to force caching
+        # proxy_ignore_headers X-Accel-Expires Expires Cache-Control;
+    }
+
+}
+
+From an installed NGINX web server (from a package install)
+        sendfile on;
+        tcp_nopush on;
+        tcp_nodelay on;
+        keepalive_timeout 65;
+        types_hash_max_size 2048;
+   
+
+        include /etc/nginx/mime.types;
+        default_type application/octet-stream;
+
+        ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # Dropping SSLv3, ref: POODLE
+        ssl_prefer_server_ciphers on;
+
+For reference here is the Open Knowledge Docker NGINX config options
+--plugins http,python,gevent				# obvious plugins
+--socket /tmp/uwsgi.sock 					# use a socket for communication
+--uid 92 								# specific User ID for the process
+--gid 92 								# Specific Group ID for the process
+--http :5000 							# Listening on port 5000 (http protocol)
+--master 								# built-in prefork+threading multi-worker management mode
+--enable-threads 							# maintain Python threads support without starting multiple threads for your application
+--paste config:/srv/app/production.ini 	# —paste is a “paste compatible framework” like pylons
+--paste-logger 
+--lazy-apps 								# will take longer to load but will be more consistent (load per worker, rather than load to first worker then fork copies)
+--gevent 2000 							# gevent is a plugin, ( 2000 is the number of async cores to spawn)
+-p 2 
+-L
+
+-------------------------------------------Simple Flaskapp and NGINX/uWSGI install---------------------------------------
+
+Deploying a source install (NGINX plus uWSGI steps to be added)
+
+The CKAN team have consolidated the various Web Server installation instructions into one configuration which we think will satisfy the majority of user needs: NGINX with uWSGI
+This guide explains how to deploy a simple Flask-based application using Nginx and uWSGI on an Ubuntu server. 
+These instructions have been tested on Ubuntu 18.04.
+The NGINX Web Server communicates to a uWSGI Server through a Linux socket file. All client HTTP requests are simply passed through to an application entry point via this socket - this entry point will then call the application - a proper diagram will follow :-
+￼
+This assumes you have installed the following packages:
+sudo apt install python3-pip python3-dev build-essential libssl-dev libffi-dev python3-setuptools python3-venv
+
+The application root is: /path/to/flaskapp ($APPROOT)
+
+1. Install Nginx
+Install Nginx (a web server) which will proxy the content from Apache and add a layer of caching:
+sudo apt install nginx
+sudo ufw allow ’Nginx HTTP’ (ufw = ubuntu firewall)
```

Comment:
In these docs, do we want to get into [firewall settings](https://help.ubuntu.com/community/UFW)? What about just recommending for a _"true production setup, it's recommended to harden your server for improved security"_?